Sample/Sample.csproj

```diff
@@ -106,10 +106,10 @@
     <Folder Include="Resources\drawable\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Android.Support.Design" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Android.Support.CustomTabs" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.3.1" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.7.0" />
+    <PackageReference Include="Xamarin.AndroidX.AppCompat">
```

Comment:
if you like you can use following syntax
<PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.3.1.1" />

Reply:
When VS updates versions keeps the current syntax but by adding new package breaks it.
Its an OCD to prefer the shorter :D Thanks.

Reply:
Done 👍